src/Klarna/Rest/Resource.php

```diff
@@ -253,7 +269,12 @@ protected function patch($url, array $data)
      */
     protected function put($url, array $data)
     {
-        return $this->request('PUT', $url, ['Content-Type' => 'application/json'], json_encode($data));
+        return $this->request(
+            'PUT',
```

Comment:
Use constant from Method Class

Reply:
Replaced all the text methods in all test files.